template/drush/README.md

```diff
@@ -1,3 +1,13 @@
+# Drush configuration and aliases
+
 This directory is intended to contain drush configuration that is not site or environment specific.
 
 Site specific drush configuration lives in `sites/[site-name]`.
+
+## Site aliases
+
+You should add the Drush aliases for any remote environments (such as Acquia Cloud) to the `site-aliases` directory. This allows developers to access remote environments using simple aliases such as `drush @mysite.dev uli`. Note that if you are using Acquia Cloud, developers can also download these aliases manually from their Insight account, but providing them with the project makes everyone’s life a little easier.
```

Comment:
Would the instructions for that basically consist of "Copy example.local.aliases.drushrc.php to local.aliases.drushrc.php and insert the appropriate local values"?

It seems like that's something Bolt could do as part of the setup task, since it already knows your project name. The 'root' would vary for each developer, but Bolt could probably figure out the correct value for that pretty easily. Probably want to open a separate PR for something that extensive.